sdk/monitor/monitor-opentelemetry/src/logs/handler.ts

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import {
+  LoggerProvider,
+  LoggerConfig,
+  SimpleLogRecordProcessor,
+  Logger as OtelLogger,
+  ConsoleLogRecordExporter,
+} from "@opentelemetry/sdk-logs";
+import { LoggerProviderConfig } from "@opentelemetry/sdk-logs/build/src/types";
+import { AzureMonitorOpenTelemetryConfig } from "../shared/config";
+import { MetricHandler } from "../metrics/handler";
+
+/**
+ * Azure Monitor OpenTelemetry Log Handler
+ */
+export class LogHandler {
+  private _loggerProvider: LoggerProvider;
+  private _logger: OtelLogger;
+  private _exporter: ConsoleLogRecordExporter;
+  private _logRecordProcessor: SimpleLogRecordProcessor;
+  private _config: AzureMonitorOpenTelemetryConfig;
+  private _metricHandler?: MetricHandler;
+
+  /**
+   * Initializes a new instance of the TraceHandler class.
+   * @param _config - Distro configuration.
+   * @param _metricHandler - MetricHandler.
+   */
+  constructor(config: AzureMonitorOpenTelemetryConfig, metricHandler?: MetricHandler) {
+    this._config = config;
+    this._metricHandler = metricHandler;
+    const loggerProviderConfig: LoggerProviderConfig = {
+      resource: this._config.resource,
+    };
+    this._loggerProvider = new LoggerProvider(loggerProviderConfig);
+    this._exporter = new ConsoleLogRecordExporter();
+    this._logRecordProcessor = new SimpleLogRecordProcessor(this._exporter);
+    this._loggerProvider.addLogRecordProcessor(this._logRecordProcessor);
+    const loggerConfig: LoggerConfig = {
+      includeTraceContext: true,
+    };
+    this._logger = this._loggerProvider.getLogger(
+      "AzureMonitorLogger",
+      undefined,
+      loggerConfig
+    ) as OtelLogger;
+    if (this._metricHandler) {
+      // TODO: Use metric handler to track standard metrics
```

Comment:
Do we need an `if` statement here if it's all TODO?  When is this work expected to be done?